### DISCUSSION TITLE
Integrating Appium/WebdriverIO test results

### DISCUSSION THREAD

**Discussion by Maduranga** [sorry-cypress/sorry-cypress#512] [Ideas]
First of all, awesome project.
Absolutely love it.

We will use Sorry-Cypress for record all Cypress test runs.
We also do testing on our mobile app with Appium/WebDriverIO.
Just thinking it would be amazing if we can also have Appium test results on the same dashboard.

**Comment by agoldis** [comment]
That's an interesting feature, do you have any idea / experience with those tools?
